src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -154,29 +154,34 @@ private static Optional<String> dumpMapReduceCounters() {
      If, for any reason, this becomes an issue in the future,
      use {@link org.apache.hadoop.mapreduce.Counters}
     */
-    List<Counters> list = SessionState.get()
-        .getMapRedStats().values().stream()
-        .map(MapRedStats::getCounters)
-        .collect(Collectors.toList());
+    List<Counters> list =
+        SessionState.get().getMapRedStats().values().stream()
+            .map(MapRedStats::getCounters)
+            .collect(Collectors.toList());
     return generateCountersJson(list, c -> c.getName(), c -> c.getValue(), Group::getDisplayName);
   }
 
   private static Optional<String> dumpTezCounters(QueryPlan plan) {
     List<TezTask> tezTasks = Utilities.getTezTasks(plan.getRootTasks());
-    List<TezCounters> list = tezTasks.stream().map(TezTask::getTezCounters).collect(Collectors.toList());
-    return generateCountersJson(list, TezCounter::getName, TezCounter::getValue,CounterGroup::getDisplayName);
+    List<TezCounters> list =
+        tezTasks.stream().map(TezTask::getTezCounters).collect(Collectors.toList());
+    return generateCountersJson(
+        list, TezCounter::getName, TezCounter::getValue, CounterGroup::getDisplayName);
   }
 
   /**
-   * Counters are deeply nested sets of key - value pairs. This attempts to dump them as
-   * is, preserving their original structure.
+   * Counters are deeply nested sets of key - value pairs. This attempts to dump them as is,
+   * preserving their original structure.
    */
-  private static <C, G extends Iterable<C>> Optional<String> generateCountersJson(List<? extends Iterable<G>> list,
-      Function<C, String> nameFn, Function<C, Long> valueFn, Function<G, String> displayNameFn) {
+  private static <C, G extends Iterable<C>> Optional<String> generateCountersJson(
+      List<? extends Iterable<G>> list,
+      Function<C, String> nameFn,
+      Function<C, Long> valueFn,
+      Function<G, String> displayNameFn) {
     JSONArray outerObj = new JSONArray();
 
-    list
-        .forEach(counters -> {
+    list.forEach(
+        counters -> {
```

Comment:
Optional: I would suggest to move the formatting changes to a separate PR (possibly reformatting the whole codebase), especially since there are many such changes in this PR and they mix with the actual feature changes.

Reply:
Created #29